neurots/preprocess/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""Functions used for preprocessing."""
+from collections import defaultdict
+
+from neurots.generate.algorithms import tmdgrower
+from neurots.validator import validate_neuron_distribs
+from neurots.validator import validate_neuron_params
+
+_preprocess_functions = defaultdict(set)
```

Comment:
Minor: I would capitalize this as a global entity.

Reply:
And please don't add code to `__init__.py` modules if it can be avoided.

Reply:
Yeah, it was just a draft but I agree I will reorganize this.